docs/user-docs/latest/content-types-builder/configuring-fields-content-type.md

```diff
@@ -0,0 +1,406 @@
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+| Select allowed types of media  | Click on the drop-down list to untick media types not allowed for this field. |
+
+:::
+
+::::
+
+### JSON
+
+The JSON field allows to configure data in a JSON format. <!--- More info needed for this field --->
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name  | Instructions                                                    |
+|---------------|-----------------------------------------------------------------|
+| Name          | Write the name of the JSON field.                               |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Unique field   | Tick to prevent another field to be identical to this one.                  |
+| Maximum length | Tick to define a maximum number of characters allowed.                      |
+| Minimum length | Tick to define a minimum number of characters allowed.                      |
+
+:::
+
+::::
+
+### UID
+
+The UID field displays a field that sets a unique identifier, optionally based on an existing other field from the same content-type.
+
+:::: tabs
+
+::: tab Base settings
+
+| Setting name   | Instructions                                                    |
+|----------------|-----------------------------------------------------------------|
+| Name           | Write the name of the UID field.                                |
+| Attached field | Choose what existing field to attach to the UID field. Choose *None* to not attach any specific field. |
+
+:::
+
+::: tab Advanced settings
+
+| Setting name   | Instructions                                                                |
+|----------------|-----------------------------------------------------------------------------|
+| Default value  | Write the default value of the UID field.                                   |
+| Private field  | Tick to make the field private and prevent it from being found via the API. |
+| Required field | Tick to prevent creating or saving an entry if the field is not filled in.  |
+| Maximum length | Tick to define a maximum number of characters allowed.                      |
+| Minimum length | Tick to define a minimum number of characters allowed.                      |
+
+:::
+
+::::
+
+## Dynamic zones & Components
```

Comment:
I would love to have a call for this one.

DZ and Components are quite different in the way they work in the CTB.
I would have split them into two different sections.

Reply:
Agree with Jim :-)

With my background in using a monolithic CMS like WordPress, I'd say they _kind of_ fulfill the  use cases of what ACF ([Advanced Custom Fields](https://www.advancedcustomfields.com/), a popular plugin to extend WordPress admin UI) calls ["Groups"](https://www.advancedcustomfields.com/resources/group/) ( = our Components) and ["Flexible Content"](https://www.advancedcustomfields.com/resources/flexible-content/) ( = our Dynamic Zones).

If you need another call with me, I'd be happy to help :-)

Reply:
Thank you for your feedback & explanations! I've changed the structure to have 2 separate sections, one for components and the other for dynamic zones. I also added more explanations to make it easier to understand the difference between the two. Hopefully it's better now!